frontend/src/pages/home/Home.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { Button, Hint, HintBody, HintTitle, PageSection } from '@patternfly/react-core';
+import { TimesIcon } from '@patternfly/react-icons';
+import { useNavigate } from 'react-router-dom';
+import { useLocalStorage } from '@patternfly/quickstarts';
+import ProjectsSection from '~/pages/home/ProjectsSection';
+import OrganizeSection from '~/pages/home/OrganizeSection';
+import QuickStartsSection from '~/pages/home/QuickStartsSection';
+import EnableTeamSection from '~/pages/home/EnableTeamSection';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
+import { useAccessReview } from '~/api';
+
+const accessReviewResource: AccessReviewResourceAttributes = {
+  group: 'project.openshift.io',
+  resource: 'projectrequests',
+  verb: 'create',
+};
+
+const Home: React.FC = () => {
+  const navigate = useNavigate();
+  const [hintHidden, setHintHidden] = useLocalStorage('rhodsNewLandingPageMessage', null);
```

Comment:
Can we call this `odh.dashboard.<something>` (eg. `odh.dashboard.landing.message`) or something to match with the other ODH storage flags we have?

<img width="1016" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/8126518/c4c102af-e5c9-4f01-8dfa-6d1db9b164f0">